get-list-of-repos/README.md

```diff
@@ -0,0 +1,13 @@
+# Domain List
+
+This script is to get a list of live domains that Isomer has.
+
+Steps to get the updated list:
+
+0. Create a `.env` file and copy over contents from .env-example over + obtain the secret api key directly from 'https://app.keycdn.com/users/authSettings'
+1. Go into the db instance in the reader mode (typically done via tableplus)
+2. Enter the query `select "primary_domain_source" from launches`
+3. Export the results that are displayed in a CSV format
+4. Copy paste over the content to db-launches-table.csv
+5. Run `npm get-domains`
```

Comment:
Should this be `node get-list-of-domains.js`?

Reply:
7a68ad441e3b9eb3a2b653cea8bc1a869e474c35 typo 
`npm run get-domains` runs `node get-list-of-domains.js`